src/Symfony/Component/Cache/Adapter/ArrayAdapter.php

```diff
@@ -97,7 +112,11 @@ public function hasItem($key)
     public function getItem($key)
     {
         if (!$isHit = $this->hasItem($key)) {
-            $this->values[$key] = $value = null;
+            $value = null;
+
+            if (!$this->maxItems) {
```

Comment:
So if no maxItems, we keep track of misses, but not if it is enabled? Anyway we can align that to simplify? Or if not maybe add inline not on why this is here.

Reply:
on purpose, comment added